README.md

```diff
@@ -181,49 +184,85 @@ terminal or deactivate the Sherpa environment.
 Source Build
 ------------
 
-
-### 2. Build from source tarball
-
-If you donwloaded the Sherpa source tarball, you can build it by:
-
-    $ tar xf sherpa-<version>.tar.gz
-    $ cd sherpa-<version>
-    $ python setup.py install
-
-Prerequisites:
+The prerequisites for building from source are:
 
  - Python: `setuptools`, `numpy`
  - System: `gcc`, `g++`, `gfortran`, `make`, `flex`, `bison`
 
+At present it is suggested that Sherpa be built with Python 2.7.
```

Comment:
I don't think Sherpa builds with Python 3.x, and last time I tried (by mistake) to build Sherpa with Python 2.6 it failed. Sherpa is not supported with these versions. So, shouldn't this statement about the Python version somewhat stronger?